README.md

```diff
@@ -19,7 +19,7 @@ import App from './App';
 import FullStory from 'TBD/package-to-be-deployed';
 
 
-FullStory.init('<your org id here>');
+FullStory.init({ orgId: '<your org id here>' });
 
```

Comment:
This is just a quick README fixup I'm sneaking in with the PR. It's not germane to the CI updates.